frontend/src/metabase/redux/downloading-analytics.ts

```diff
@@ -0,0 +1,23 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+
+import type { ResourceAccessedVia, ResourceType } from "./downloads";
+
+const SCHEMA_VERSION = "1-0-0";
+const SCHEMA = "downloads";
+
+export const trackDownloadResults = ({
```

Comment:
Should we move this outside of the `metabase/redux` directory, as this is more of a utility function than a Redux action?

Reply:
When I wrote this feature initially it was not in redux, some changes on master moved the logic file here and i just moved the analytics file as we usually keep it close to where we use it.
Do you have suggestion for where to put it?

(Meanwhile, I'll rename it to downloads-analytics, as the logic file also got renamed in master from downloading to downloads)